pkg/webui/console/containers/event-split-frame/event-split-frame.styl

```diff
@@ -21,6 +21,7 @@
   flex-direction: column
   z-index: $zi.nav
   position: relative
+  max-height: "calc(100vh - %s)" % ($header-height)
```

Comment:
This was just a top of my head suggestion. `$header-height` is not actually used currently in the header. So you'll need to update that as well.

Reply:
I saw it is set in the `generic.styl` so that's why I used it

Reply:
Yeah but that's a relic from the old Console design.

Reply:
Updated now